Cargo.toml

```diff
@@ -45,3 +47,10 @@ serde = ["serdect"]
 [package.metadata.docs.rs]
 all-features = true
 rustdoc-args = ["--cfg", "docsrs"]
+
+[[bench]]
+name = "bench"
+harness = false
+
+[profile.bench]
+opt-level = 3
```

Comment:
FWIW I think this is the default for the bench profile (which is the same as the `release` profile)

Reply:
Confirmed: https://doc.rust-lang.org/cargo/reference/profiles.html#bench

> The `bench` profile inherits the settings from the `release` profile.

```
[profile.release]
opt-level = 3
```

Reply:
Fixed.